### DISCUSSION TITLE
Has hisense A7CC successfully flashed the gsi system? xdaforums.com has your reply method to try.

### DISCUSSION THREAD

**Discussion by lxb158** [4bitFox/hisense_a7cc#1] [General]
Has hisense A7CC successfully flashed the gsi system? xdaforums.com has your reply method to try.

**Comment by 4bitFox** [comment]
No I haven't managed to get any GSI to boot unfortunately :-(

https://github.com/4bitFox/hisense_a7cc/blob/main/README.md#gsi
